__tests__/app.test.js

```diff
@@ -62,26 +62,91 @@ describe("GET ./api/topics", () => {
 });
 
 describe("GET /api", () => {
-    test("response code is 200", () => {
-        return request(app)
-        .get("/api")
-        .expect(200);
+  test("response code is 200", () => {
+    return request(app).get("/api").expect(200);
+  });
+
+  test("returns an object", () => {
+    return request(app)
+      .get("/api")
+      .expect(200)
+      .then((response) => {
+        expect(typeof response.body).toEqual("object");
+      });
+  });
+  test("returns value of endponts.json file", () => {
+    return request(app)
+      .get("/api")
+      .expect(200)
+      .then((response) => {
+        expect(response.body).toEqual(endpointsJson);
+      });
+  });
+});
+describe("GET /api/articles/:article_id", () => {
+  test("response code is 200", () => {
+    return request(app).get("/api/articles/1").expect(200);
+  });
+  test("returns an object", () => {
+    return request(app)
+      .get("/api/articles/1")
+      .then((response) => {
+        expect(typeof response.body.article).toBe("object");
+        expect(Array.isArray(response.body.article)).toBe(false);
+        expect(response.body.article).not.toBeNull();
       });
+  });
+  test("returns correct article first object", () => {
+    const expected = {
+      article_id: 1,
```

Comment:
We need to be careful testing for object shapes in this way. If we were to add a column to our article object, we would have to reflect this in every test. It would be better to check that the specific columns we know exist are currently there. That way, if we were to add another column later one, we could test for that separately.